index.html

```diff
@@ -22,7 +22,7 @@
     <link rel="icon" type="image/png" sizes="16x16" href="/public/icons/favicon-16x16.png">
 
     <meta name="viewport" content="width=device-width, initial-scale=1.0" />
-    <title>Portfolio</title>
```

Comment:
Consider updating the title to reflect the content of the page for better SEO and user experience.

For reference, see [Google SEO Guide: Titles](https://developers.google.com/search/docs/advanced/appearance/title-snippet).